Code/Framework/AzCore/AzCore/Settings/ConfigParser.cpp

```diff
@@ -196,6 +196,11 @@ namespace AZ::Settings
                 memmove(configBuffer.begin(), frontIter, readOffset);
             }
 
+            if (remainingFileLength==0)
```

Comment:
Missing whitespace around ==.
```suggestion
            if (remainingFileLength == 0)
```